src/controllers/sync-controller.ts

```diff
@@ -99,30 +322,25 @@ export class SyncController implements ReactiveController {
       };
       navigator.serviceWorker.addEventListener('message', this.#messageHandler);
 
-      // Check if periodic sync should be enabled based on settings
-      const settings = this.settings;
-      if (settings?.auto_sync) {
-        this.postToServiceWorker(SyncMessages.registerPeriodicSync(true));
-      }
+      // Phase 1.3: Log capability status for debugging
+      DebugService.logInfo('sync', 'Service worker ready with capabilities', {
+        backgroundSync: this.#pwaCapabilities.backgroundSync,
+        periodicBackgroundSync: this.#pwaCapabilities.periodicBackgroundSync,
+        webLocks: this.#pwaCapabilities.webLocks
+      });
+
+      // Note: Periodic sync state is now managed globally by PageVisibilityService
+      // based on page visibility to prevent service worker blocking when app is visible
     } catch (error) {
       DebugService.logSyncError(error instanceof Error ? error : new Error(String(error)), { context: 'Service worker setup failed' });
-      this.#notifyError('Failed to initialize sync service');
+      this.#notifyError('Unable to set up background sync. Please refresh the page and try again.');
     }
   }
 
   private async checkSyncStatus() {
```

Comment:
It looks like this method is just a wrapper around `this.#checkAndResumeSync()`. Combine the two methods.

Reply:
The `#checkAndResumeSync` name was more descriptive of what this method now does. Let's use that name.